mcu_cfg.hjson

```diff
@@ -28,9 +28,14 @@
         }
     },
 
-    slow_memory: {
+    ext_peripherals: {
```

Comment:
in my opinion, the peripheral should be under _**peripherals**_

like uart and soc_ctrl